tests/test_spade_vaegan.py

```diff
@@ -69,7 +74,7 @@ def create_semantic_data(shape: list, semantic_regions: int):
     return out_label_.unsqueeze(0), out_image.unsqueeze(0).unsqueeze(0)
 
 
-class TestDiffusionModelUNet2D(unittest.TestCase):
+class TestSpadeNet(unittest.TestCase):
```

Comment:
Do we need to add a `load_old_state_dict` function for this network?
cc @ericspod @marksgraham

Reply:
This being a test network we can regenerate weights for it without worrying about user compatibility.

Reply:
We decided on not doing a old state dict loader for this network considering its use.